react/features/prejoin/components/Prejoin.web.js

```diff
@@ -382,7 +372,6 @@ class Prejoin extends Component<Props, State> {
                     data-testid = 'prejoin.screen'>
                     {this.showDisplayNameField ? (<InputField
                         autoComplete = { 'name' }
-                        autoFocus = { !isAuthInProgress }
```

Comment:
Shouldn't it be true so the user doesn't need to click the field the first time around?